.changeset/two-boats-love.md

```diff
@@ -0,0 +1,5 @@
+---
+"ilib-loctool-json": patch
```

Comment:
This is a breaking change because people may have to update their project.json files to use the new version of this plugin.